Doc/library/threading.rst

```diff
@@ -989,7 +989,7 @@ method.  The :meth:`~Event.wait` method blocks until the flag is true.
 
       When the timeout argument is present and not ``None``, it should be a
       floating point number specifying a timeout for the operation in seconds
-      (or fractions thereof).
+      (or fractions thereof). When the timeout occurs, ``False`` will be returned.
```

Comment:
It is already implicitly documented in the following paragraph. If it is not clear, it would be better to rewrite that paragraph.

Reply:
I've added 2 commits. The first is as discussed. The second strips down some unnecessary and confusing description (removing the except clause from the sentence)